test/cppcheck_suppressions.txt

```diff
@@ -21,7 +21,12 @@ shiftNegative:src/utils/msc_tree.cc
 
 // 
 // ModSecurity v3 code...
-// 
+//
+
+functionConst:src/utils/shared_files.h:60
```

Comment:
Generally: please don't use this file to collect suppressions. If you need to add some suppression, please use an inline one.

But here: why it this necessary? It seems in that file in line [60](https://github.com/owasp-modsecurity/ModSecurity/blob/b0a56941fab7bc50bd13203dc11810231392e797/src/utils/shared_files.h#L60) there is only a condition...